site/src/pages/api/users/[shortname]/blocks/index.api.ts

```diff
@@ -0,0 +1,35 @@
+import { createBaseHandler } from "../../../../../lib/handler/baseHandler";
+import { User } from "../../../../../lib/model/user.model";
+import { formatErrors } from "../../../../../util/api";
+import { ExpandedBlockMetadata } from "../../../../../lib/blocks";
+
+export type ApiBlocksByUserResponseQueryParams = {
+  shortname: string;
+};
+
+export type ApiBlocksByUserResponseResponse = {
```

Comment:
Sorry I'm dropping this after approving. While integrating, I noticed in some places we use the format `ApiXYZResponse`, while in other places we use `ApiXYZResponseResponse`. Not sure why the latter has two `Response`, but I think we should stick to 1 pattern

Reply:
Thanks @teenoh - this is just a typo, I'll remove it

Reply:
Have renamed this in the next PR (on the frontend UI)